kitsune/users/models.py

```diff
@@ -69,6 +69,10 @@ class Profile(ModelBase, SearchMixin):
                             verbose_name=_lazy(u'City'))
     locale = LocaleField(default=settings.LANGUAGE_CODE,
                          verbose_name=_lazy(u'Preferred language'))
+    first_answer_email_sent = models.BooleanField(
+        default=False, verbose_name=_lazy('Has been sent a first answer contribution email.'))
+    first_l10n_email_sent = models.BooleanField(
+        default=False, verbose_name=_lazy('Has been sent a first revision contribution email.'))
```

Comment:
I honestly don't remember writing that. I guess past-me copied it from another sumo field, and then forgot about it. I don't have a lot of experience with the admin or configuring it. Do you think I should switch this to `help_text` instead of `verbose_name`?